index.js

```diff
@@ -18,19 +18,22 @@ module.exports = {
   included() {
     this._super.included.apply(this, arguments);
 
-    const host = this._findHost();
-    this.addonConfig = host.options['apollo'] || {};
+    this.host = this._findHost();
```

Comment:
Is it necessary to store this reference to the host, rather than just calling `_findHost()` in the `addonConfig` method?

Reply:
I don't have strong feelings on this. `_findHost()` does a little bit of work, iterating up the addon tree (`while (parent = this.parent)`). But this code path is called infrequently so the cost of the work won't add up to much.

What do you think?

Reply:
That's true! I don't have a strong argument either way, your point about going up the tree is really good.